iOS/Projects/Features/Record/Sources/ExerciseSelectScene/View/ExerciseSelectViewController.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  ExerciseSelectViewController.swift
+//  RecordFeature
+//
+//  Created by MaraMincho on 11/16/23.
+//  Copyright © 2023 kr.codesquad.boostcamp8. All rights reserved.
+//
+
+import DesignSystem
+import UIKit
+
+// MARK: - ExerciseSelectViewController
+
+final class ExerciseSelectViewController: UIViewController {
+  override init(nibName _: String?, bundle _: Bundle?) {
+    super.init(nibName: nil, bundle: nil)
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    setupConstraints()
+    navigationController?.setNavigationBarHidden(true, animated: false)
+  }
+
+  required init?(coder: NSCoder) {
+    super.init(coder: coder)
+  }
```

Comment:
p2: 사용하지 않는다면 제거하는 편이 깔끔할 것 같아요.
```suggestion
  override func viewDidLoad() {
    super.viewDidLoad()
    setupConstraints()
    navigationController?.setNavigationBarHidden(true, animated: false)
  }
```

Reply:
넵

현재 코드에서는, 둘이 들어갈 필요가 없어 보입니다.